### ISSUE TITLE
Create MAC Design Guideline

### ISSUE THREAD

**Issue by khanguslee** [monashcode/mac-website#4]
To make sure all our projects conform to the same colour scheme etc. we will need to get marketing to create this for us!

**Comment by khanguslee** [comment]
Some inspiration:

- [MongoDB](http://mongodb.design/#/ui-design-system/base-styles/colors)
- [Autodesk](http://brand.autodesk.com/brand-elements/colors)
- [Google](https://material.io/design/color/the-color-system.html#color-usage-palettes)
- [Window Apps](https://docs.microsoft.com/en-us/windows/uwp/design/style/color)
- [Spotify](https://developer.spotify.com/branding-guidelines/)
- [Atlassian](https://www.atlassian.design/guidelines/brand/color)

**Comment by aprilxyc** [comment]
Jasmine's first project on MAC was a style guide so it has already been made here: https://drive.google.com/file/d/1c2a8hDsFiA0YG6SItU5ryKmPBIR7SpKP/view?usp=sharing. It probably just needs to be refined for the website but is a good starting point for now!

**Comment by khanguslee** [comment]
If the technical team does finish setting up stuff before we get designs for the actual MAC website, we can possibly start creating a simple website that just displays all of these MAC design guidelines for now. (Can be redesigned later on)

**Comment by aprilxyc** [comment]
I agree that would be an awesome starting project! + based on those design guidelines you linked, I reckon react would be super good for a simple website like that
